SU-SWS/stanford_paragraph_types
# Earth 16 fw banner

# READY FOR REVIEW

# Summary
- Created and display full width banner paragraph items.

# Needed By (Date)
- End of sprint

# Urgency
- Low urgency

# Steps to Test

1. checkout branch & https://github.com/SU-SWS/stanford_components/pull/26
2. `drush en stanford_paragraph_fw_banner`
3. `drush cr`
4. create a complex page with a Full width banner
5. Verify the full width banner looks mostly accurate. (I think the text/link color needs some changes)

# See Also
- https://github.com/SU-SWS/stanford_components/pull/26
